django/contrib/admin/checks.py

```diff
@@ -944,6 +945,24 @@ def _check_action_permission_methods(self, obj):
                     )
         return errors
 
+    def _check_actions_uniqueness(self, obj):
+        """ Check that every action has a unique __name__. """
+        actions = obj._get_base_actions()
+        names = [action[1] for action in actions]
+
+        if len(names) != len(set(names)):
+            return [
+                checks.Error(
+
+                    '__name__ attributes of actions defined in %s must be unique.' % (
+                        obj.__class__.__name__
```

Comment:
Would `obj.__class__` not be more helpful? (That way I get the full import path, rather than just the name, which may not be unique.)